docker/data-science/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN python -m pip --no-cache-dir install --upgrade pip && \
 
 RUN python -m spacy download en_core_web_sm
 RUN pip --no-cache-dir install https://github.com/explosion/spacy-experimental/releases/download/v0.6.1/en_coreference_web_trf-3.4.0a2-py3-none-any.whl
+RUN pip install git+https://github.com/openai/CLIP.git
```

Comment:
was the git version needed due to a bug fix? I'm going to try just pulling public release artifacts (1.0.1) to see if it works if not.

Reply:
I've moved this into requirements.txt